themes/bootstrap3/templates/admin/menu.phtml

```diff
@@ -5,7 +5,5 @@
   <a href="<?=$this->url('admin/maintenance')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'maintenance' ? ' active' : ''?>"><?=$this->transEsc('System Maintenance')?></a>
   <a href="<?=$this->url('admin/tags')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'tags' ? ' active' : ''?>"><?=$this->transEsc('Tag Maintenance')?></a>
   <a href="<?=$this->url('admin/feedback')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'feedback' ? ' active' : ''?>"><?=$this->transEsc('Feedback Management')?></a>
-  <?php if ($this->overdrive()->showOverdriveAdminLink()):?>
-    <a href="<?=$this->url('admin/overdrive')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'overdrive' ? ' active' : ''?>"><?=$this->transEsc('Overdrive API')?></a>
-  <?php endif;?>
+  <a href="<?=$this->url('admin/overdrive')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == 'overdrive' ? ' active' : ''?>"><?=$this->transEsc('Overdrive API')?></a>
```

Comment:
I don't think the string Overdrive API exists in en.ini -- and shouldn't it be OverDrive API in any case?